22-discovery.ks

```diff
@@ -15,6 +15,9 @@ systemctl enable NetworkManager-wait-online.service
 echo " * enabling nm-prepare service"
 systemctl enable nm-prepare.service
 
+echo " * enabling nm-connected service"
+systemctl enable nm-connected.service
+
```

Comment:
Hmmm I thought that since `nm-connected` have `Before=network-online.target` we don't need that.